packages/blocks/stopwatch/src/app.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState, useCallback } from "react";
+import { BlockComponent } from "blockprotocol/react";
+
+type AppProps = {
+  start: Date;
+  laps: number[];
+};
+
+function formatDuration(duration: number) {
```

Comment:
```suggestion
const formatDuration = (duration: number) => {
```

We tend to use arrow fn style where possible for consistency. This is an optional suggestion 🙂

Reply:
Consistency is good!